src/app/my-account/my-account.component.html

```diff
@@ -0,0 +1,54 @@
+<div class="center">
+  <h1>My Account</h1>
+  <form style="width: 75%; display: inline-block">
+    <div [ngClass]="{
+      blue: passColors.oldPass === this.colors.blue,
+      green: passColors.oldPass === this.colors.green,
+      red: passColors.oldPass === this.colors.red,
+      group: true
+    }">
+      <label for="oldPassword" class="group-start">Old Password:</label>
+      <input [(ngModel)]="passFields.oldPassField" (input)="oldPassChange()" type="password" id="oldPassword" class="group-end" name="oldPassword" placeholder="Old Password...">
+    </div>
+    
+    <div>
+      <h2>A strong password satisfies the following criteria:</h2>
```

Comment:
Do these requirements come from Candice? If possible I'd like to encourage users to use passphrases (eg. https://xkcd.com/936 ), which are superior to passwords.